### ISSUE TITLE
Cannot sign in to Apple Music

### ISSUE THREAD

**Issue by willnotwash** [ubports/morph-browser#301]
Apple launched a web-based version of their music service last year at https://beta.music.apple.com. This is a great way to access this service on Linux device, in the absence of a native Linux client, and would probably lend itself nicely to a UT web app. However it doesn't work on Morph Browser because you can't sign into the service. When you access the web site, the sign-in button is at the top right of the page. When you press this it should open another browser tab allowing you to sign in. This works on Linux desktop using Firefox etc. However, in Morph it just opens another default 'about:blank' tab. I wonder if this issue might be related to issue number 295: “Webapps do not open links from url dispatcher #295“. 

It would be great to be able to access Apple Music from UT. 

I'm using a Nexus 7 flo and this seems to happen in both the stable and RC channels.

Thanks for reading, and for all the tremendous work on UT being done here.

**Comment by gbdomubpkm** [comment]
It seems to be working now with your link. You can check that.

**Comment by willnotwash** [comment]
Yes I just checked it there and sign in is now working. Apple seems to have changed the way sign in is done and it now works, thanks. However, I still cannot actually play any music using the Morph browser. When you go into your Apple Music library, click on "Songs" and then select any song to play, it says "Failed to load because no supported source was found". It's a pity, as this would allow me to switch to UBPorts and retain access to my Apple Music library.
